### PR TITLE
Switch chaos base image to ubi

### PR DESCRIPTION
**Pull request by chaitanyaenr**

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 base_images:
   os:
-    name: centos
+    name: ubi
     namespace: openshift
-    tag: stream9
+    tag: latest
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:5]: this should be 8 I believe

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 base_images:
   os:
-    name: centos
+    name: ubi
     namespace: openshift
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:4]: think this needs to be ocp

Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml]: Fixed. Thanks.

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml (hunk 1 above)
Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml]: Fixed. Thanks.

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 base_images:
   os:
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml:2]: think this should be krkn to be consistent with what the base image is

**Comment by chaitanyaenr** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml (hunk 3 above)
Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main.yaml]: Done!

**Pull request closed without merging**